### PR TITLE
[openstack] fix openstack masked service check

### PR DESCRIPTION
**Pull request by dosaboy**
Adds support for registering services that are expected to
be marked as masked with each project definition and excludes
them from the final check.

Resolves: #183

### CODE REVIEW

File: core/checks.py [hunk 1]
```diff
@@ -971,7 +971,7 @@ def services(self):
                 # make it a service
                 # NOTE: the @ is left out from the service name as it implies
```
Comment (nicolasbock) [inline core/checks.py:972]: Update the documentation here as well:
```suggestion
                # NOTE: the @ is include in the service name as it implies
```

**Review by nicolasbock — APPROVED** [review]
LGTM. Maybe two small documentation updates.

File: core/checks.py [hunk 2]
```diff
@@ -971,7 +971,7 @@ def services(self):
...
                 # this is an "indirect" service that has a .target.
```
[the remark this answers is not in the document]
Comment (dosaboy) [reply core/checks.py:973]: this has made me think twice and im gonna have a rework to make it clearer

File: plugins/openstack/pyparts/service_info.py [hunk 3]
```diff
@@ -33,9 +33,8 @@ def get_masked_services(self):
         """Get a list of masked services."""
         if self.service_info_str['systemd']:
             masked = self.service_info_str['systemd'].get('masked', {})
-            if 'keystone' in masked:
-                del masked['keystone']
-
+            expected_masked = OST_PROJECTS.default_masked_services
+            masked = set(masked).difference(expected_masked)
```
Comment (nicolasbock) [inline plugins/openstack/pyparts/service_info.py:37]: Nice use of `set` :smile:

**Review by nicolasbock — APPROVED** [review]
LGTM

**Review by nicolasbock — APPROVED** [review]

**Pull request merged**